Prefabs/Player.prefab

```diff
@@ -580,80 +580,6 @@
                         }
                     }
                 },
-                "Component_[9177231166678061715]": {
-                    "$type": "EditorScriptCanvasComponent",
-                    "Id": 9177231166678061715,
-                    "configuration": {
-                        "sourceHandle": {
-                            "id": "{A79F36FD-74EB-5C3A-A188-7A8E467FF599}",
-                            "path": "scriptcanvas/SpawnAndAttachLazerGun.scriptcanvas"
```

Comment:
want to delete the script canvas file too since it doesn't work anyways? can also bring it back using git.